Example/HorizonCalendarExample/HorizonCalendarExample/FooterView.swift

```diff
@@ -0,0 +1,48 @@
+//  Created by Sebastien Audeon on 11/26/20.
+//  Copyright © 2020 Airbnb. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import HorizonCalendar
+import UIKit
+
+// MARK: CalendarItemViewRepresentable
```

Comment:
```suggestion
// MARK: - FooterView
```

Reply:
And for consistency, add the `CalendarItemViewRepresentable` conformance below so that it looks similar to the `DayView` file structure.